<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-B4RJ235" failures="1" tests="3" name="Test" time="28.037" errors="0" timestamp="2024-07-12T12:17:03 IST">
  <testcase classname="com.listeners.demo.OrangeHrm" name="testLogo" time="0.443"/>
  <testcase classname="com.listeners.demo.OrangeHrm" name="testUrl" time="0.058">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at com.listeners.demo.OrangeHrm.testUrl(OrangeHrm.java:55)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testUrl -->
  <testcase classname="com.listeners.demo.OrangeHrm" name="testTitle" time="0.0">
    <skipped/>
  </testcase> <!-- testTitle -->
</testsuite> <!-- Test -->
